lib/ruote/dashboard.rb

```diff
@@ -1363,8 +1363,6 @@ def initialize(dashboard, clear)
 
       @dashboard = dashboard
 
-      @dashboard.context.plist.clear if clear
```

Comment:
I wrote the test in the previous commit.  No tests failed before I wrote
the TDD commit, none failed after I made the change.

Rick Cobb / Architect

Grand Rounds

Office: 1-800-929-0926  / Fax: 1-888-323-5997

360 3rd St, Suite 425,  San Francisco, CA 94107

http://www.grandroundshealth.com ​

[image: Facebook] <https://www.facebook.com/GrandRoundsHealth> [image:
Twitter] <https://twitter.com/GrandRoundsInc> [image: Linkedin]
<https://www.linkedin.com/company/grandrounds>

On Tue, Jun 14, 2016 at 4:02 PM, Andrew Purcell <notifications@github.com>
wrote:

> In lib/ruote/dashboard.rb
> <https://github.com/ConsultingMD/ruote/pull/4#discussion_r67073675>:
>
> > @@ -1363,8 +1363,6 @@ def initialize(dashboard, clear)
> >
> >        @dashboard = dashboard
> >
> > -      @dashboard.context.plist.clear if clear
>
> That's it? No side effects or failing tests?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ConsultingMD/ruote/pull/4/files/062463f7ab7b8fc1c47817ddfc84a841d63f8729#r67073675>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAPN1bz1w4_BzoOR_13Hdc_s4YyJamNVks5qLzL5gaJpZM4I1zv8>
> .
>